community/brillo/depends

```diff
@@ -0,0 +1,2 @@
+go
+go-md2man
```

Comment:
Is this needed solely for docs?

Reply:
Seems to be the case. I'd much rather we disable docs or ship a pre-generated manual page than add a dependency on Go to build something written in C.

Reply:
Seems `go-md2man` is quite useless, it serves no purpose other than bloat as a dependancy for `brillo`.
I've pushed a new commit removing it.